cfme/configure/settings.py

```diff
@@ -349,12 +349,17 @@ def __init__(self, appliance=None):
         Navigatable.__init__(self, appliance=appliance)
 
     @classmethod
-    def set_default_view(cls, button_group_name, default):
+    def save(cls):
```

Comment:
it's going to be something like:
`@classmethod
    def set_default_view(cls, button_group_names, defaults):

        if not isinstance(button_group_names, (list, tuple)):
            button_group_names = [button_group_names]
        if not isinstance(defaults, (list, tuple)):
            defaults = [defaults]*len(button_group_names)
        assert len(button_group_names) == len(defaults)

        for button_group_name, default in zip(button_group_names, defaults):
            bg = ButtonGroup(button_group_name)
            navigate_to(cls, 'All')
            if bg.active != default:
                bg.choose(default)
        sel.click(form_buttons.save)`